docs/tutorial.md

```diff
@@ -539,7 +856,7 @@ Add a new component named **MyLists**. This component obtains shopping list data
 
 **client/components/MyLists.js**
 
-```js
+```jsx
 import React, { useState, useEffect } from 'react'
 
```

Comment:
Please make the following changes to the lines below:

Line 551:
`// The 'useQuery' hook is used to querry the 'ShoppingLists' Read Model's 'all' resolver.`
"querry" is mispelled. Please fix.

Line 943:
`Currently your reSolve application can create shopping lists and their items. The application also requires the capability to toggle shopping list items as well as remove and items and entire lists.`
Please add a comma after "Currently".
What do you mean by "as well as remove and items and entire lists"?

Line 1089:
`The useCommandHook allows you send commands to reSolve.`
Please update this line as follows:
`The useCommandHook allows you to send commands to reSolve.`

Line 1101:
`// A callback user to pass info about a newly created shopping list to the parrent component.`
"parrent" is mispelled. Please fix.

Reply:
Applied your edits.
>What do you mean by "as well as remove and items and entire lists"?
Changed to "as well as remove items and entire lists"

Reply:
Thank you.